packages/bitcore-node/src/modules/firocoin/models/token.ts

```diff
@@ -13,6 +13,8 @@ export interface IToken {
   name: string;
   symbol: string;
   totalSupply: Decimal128 | string;
+  officialSite: string;
+  socialProfiles: string;
```

Comment:
should we have field price here?

Reply:
I think we could add`price?: Decimal128 | string;` and  in API we could do `token.price = token.price || "-1";`